CoreScriptsRoot/Modules/Settings/Pages/ReportAbuseMenu.lua

```diff
@@ -278,6 +278,34 @@ do
 	PageInstance.Displayed.Event:connect(function()
 		PageInstance:UpdatePlayerDropDown()
 	end)
+
+	function PageInstance:ReportPlayer(player)
+		if player then
+			local setReportPlayerConnection = nil
```

Comment:
Wouldn't it make more sense to just set the connection to `setReportPlayerConnection` in the first place instead of making it nil then assigning the connection? For example:
`local setReportPlayerConnection = PageInstance.Displayed.Event:connect(function()`. Same as on line 291.